src/js/components/tasks/table.tsx

```diff
@@ -321,11 +342,36 @@ const TaskTable = ({
           </>
         }
         property="name"
-        width="65%"
+        width={viewEpicsColumn ? '40%' : '60%'}
```

Comment:
Once this PR is done, it might be helpful for @stacyk to review these column widths. They didn't seem to have any impact as far as I could tell, and the new "Epic" column is relatively narrow.

![Screen Shot 2021-07-02 at 2 42 30 PM](https://user-images.githubusercontent.com/552316/124316596-d6533080-db43-11eb-88c0-c2a029f75202.png)

Reply:
@jgerigmeyer I added some truncation to the status column, and reduced the sizes of the other columns. There were probably some class changes that made these widths more effective, or maybe if you have different amounts of content in your app those widths respond differently. In any case, they are working as I would expect now but let me know if you see any edge cases that I didn't plan for.